saleor/order/utils.py

```diff
@@ -198,17 +199,14 @@ def recalculate_order_weight(order):
     order.save(update_fields=["weight"])
 
 
-def update_taxes_for_order_line(
+def update_order_line_prices_if_expired(
```

Comment:
I don't think that this method will be needed if we move the rest of the calculations of prices to `order.calculations` file. I believe, all of this value will be already updated as we will just call `fetch_order_prices_if_expired`.